lib/analyzer/apk-analyzer.ts

```diff
@@ -0,0 +1,74 @@
+import subProcess from '../sub-process';
+
+export {
+  analyze,
+};
+
+function analyze(targetImage) {
+  return getPackages(targetImage)
+    .then(pkgs => ({
+      Image: targetImage,
+      AnalyzeType: 'Apk',
+      Analysis: pkgs,
+    }));
+}
+
+function getPackages(targetImage) {
+  return subProcess.execute('docker', [
```

Comment:
I think that it will be better to create a new class for "docker".
Right now `subProcess.execute('docker')` is the only common function, but in the near future there will be more.
Plus, it will prevent the code dup

Reply:
Yeh, for sure. There is tons of cleanup that can be done here. Was trying to make the smallest PR possible for now :)